src/Saas.Admin/Saas.Admin.Service/Controllers/TenantDTO.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Saas.Admin.Service.Controllers;
```

Comment:
Any reason why this is located in the controllers folder and not in a models folder?

Reply:
Started there because only controller was using it, but that did change.  Makes sesne to move it, I'll do it in another PR so we can get these changes in now.